test/integration/index.js

```diff
@@ -5,18 +5,22 @@ const logger = require('./logger');
 const config = require('../knexfile');
 const fs = require('fs');
 
-const Bluebird = require('bluebird');
-
 Object.keys(config).forEach((dialectName) => {
   return require('./suite')(logger(knex(config[dialectName])));
 });
 
-after(function(done) {
+before(function() {
+  if (config.sqlite3 && config.sqlite3.connection.filename !== ':memory:') {
+    fs.copyFileSync(
```

Comment:
do we need to do the copying for the whole suite and not just one test?

Reply:
I was thinking about copying in the test body but if the test fails it will not remove the file. We could copy in the test body and delete in the teardown function but I think it's better if they are in the same place.